DataConnectors/GoogleWorkspaceReports/GWorkspaceReports_API_FunctionApp.json

```diff
@@ -0,0 +1,221 @@
+{
```

Comment:
Fix the check-in validation first:

Incorrect Json file. File path: DataConnectors/GoogleWorkspaceReports/GWorkspaceReports_API_FunctionApp.json. Error message: Unexpected token ​ in JSON at position 4213
##[error]An error occurred, please open an issue
##[error]Bash exited with code '255'.

Reply:
Fixed